server/pulp/server/db/migrations/0027_importer_schema_change.py

```diff
@@ -26,7 +26,7 @@ def migrate(*args, **kwargs):
             try:
                 importer[updated_key] = isodate.parse_datetime(importer['last_sync'])
             # The attribute doesn't exist, or parsing failed. It's safe to set a newer timestamp.
-            except:
+            except: # noqa: 722
```

Comment:
Looking at this migration, I almost think it's safe to catch `Exception` here—especially give the comment that precedes it.

Reply:
I leave it up to you though. I'm fine either way.